src/SortableList.js

```diff
@@ -111,21 +113,13 @@ export default class SortableList extends Component {
         });
       });
 
-      if (Object.keys(nextData).length > Object.keys(data).length) {
```

Comment:
It should be
`if (Object.keys(nextData).length > Object.keys(data).length || Object.keys(nextData).length < Object.keys(data).length)`